common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -330,7 +337,8 @@ def _student_view(self, context, banner_text=None):
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
             'banner_text': banner_text,
-            'disable_navigation': not self.is_user_authenticated(context),
+            'save_position': self.is_user_authenticated(context),
+            'show_completion': self.is_user_authenticated(context),
```

Comment:
`save_position` and `show_completion` should be method parameters with `True` as defaults but passed `False` from `anonymous_view()`.

Reply:
👍